public/components/trace_analytics/components/services/__tests__/service_view.test.tsx

```diff
@@ -7,39 +7,51 @@ import { configure, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import React from 'react';
 import { ServiceView } from '..';
-import { coreStartMock } from '../../../../../../test/__mocks__/coreMocks';
+
+jest.mock('../../../../../../test/__mocks__/coreMocks', () => ({
+  coreStartMock: {
+    chrome: { setBreadcrumbs: jest.fn() },
+    http: { post: jest.fn() },
```

Comment:
Can we please add mocks with some actual data (both empty and non-empty response)?

Reply:
Cypress added for service map 
Tests focus functionality in Service map
Testing the re-renders and filtering

Added 
import toJson from 'enzyme-to-json';
To service map test